docs/user/custom-resources/apirule/v2alpha1/04-10-apirule-custom-resource.md

```diff
@@ -48,7 +48,7 @@ This table lists all parameters of APIRule `v2alpha1` CRD together with their de
 | **rules.jwt.authorizations.requiredScopes**      |  **NO**  | Specifies the list of required scope values for the JWT.                                                                                                                                                                                                                              | None                                                                                                                  |
 | **rules.jwt.authorizations.audiences**           |  **NO**  | Specifies the list of audiences required for the JWT.                                                                                                                                                                                                                                 | None                                                                                                                  |
 | **rules.extAuth**                                |  **NO**  | Specifies the Istio External Authorization access strategy.                                                                                                                                                                                                                           | Must exists if noAuth and jwt are not specified.                                                                      |
-| **rules.extAuth.authorizers**                    |  **NO**  | Specifies the Istio External Authorization authorizers.                                                                                                                                                                                                                               | Validated that the provider exists in Istio external authorization providers.                                         |
+| **rules.extAuth.authorizers**                    | **YES**  | Specifies the Istio External Authorization authorizers. In case extAuth is configured, at least one needs to be present.                                                                                                                                                                                                                             | Validated that the provider exists in Istio external authorization providers.                                         |
```

Comment:
```suggestion
| **rules.extAuth.authorizers**                    | **YES**  | Specifies the Istio External Authorization authorizers. In case extAuth is configured, at least one must be present.                                                                                                                                                                                                                             | Validated that the provider exists in Istio external authorization providers.                                         |
```